scripts/release-package.sh

```diff
@@ -15,13 +24,25 @@ else
   exit
 fi
 
+if [[ -z "${changelog}" && $auto = true ]]; then
+	echo "${color_red}Skip package ${package} because changelog is empty. ${color_reset}"
+	exit
+fi
+
 echo "---------------------"
 echo "info latest release tag: ${color_blue}${last_release_tag}${color_reset}"
 echo "info latest version: ${color_blue}${last_version}${color_reset}"
 echo "release tag: ${color_blue}${release_tag}${color_reset}"
 echo -e  "Generated changelog:\n${color_blue}${changelog}${color_reset}"
 
-read -p "question New version: : " new_version
+if [[ $auto = false ]]; then
+  read -p "question New version [${next_version}] : " new_version
```

Comment:
I think this question could be a bit confusing. I would write that the value inside the square brackets is the suggested version. Also it would be nice to confirm this version without rewriting it as answer. But overwriting should still be possible.

Also the word "question" can be removed. I think this was added by me by mistake.

Reply:
If you leave the new version empty the suggested version is taken